spec/presenters/candidate_interface/application_form_presenter_spec.rb

```diff
@@ -32,6 +32,22 @@
     end
   end
 
+  describe '#contact_details_valid?' do
+    it 'returns true if contact details section is completed' do
+      application_form = FactoryBot.build(:application_form, contact_details_completed: true)
```

Comment:
To fix the tests given the above suggestion:

```suggestion
      application_form = FactoryBot.build(:completed_application_form, contact_details_completed: true)
```